src/olympia/reviewers/tests/test_serializers.py

```diff
@@ -58,8 +60,16 @@ def test_raises_without_version(self):
             serializer.data
 
     def test_can_access_version_from_parent(self):
+        # Set up the request to support drf_reverse
+        api_version = api_settings.DEFAULT_VERSION
+        request = APIRequestFactory().get('/api/%s/' % api_version)
+        request.versioning_scheme = (
+            api_settings.DEFAULT_VERSIONING_CLASS()
+        )
+        request.version = api_version
```

Comment:
Could this be done in `setUp` ? You do that below for other test classes.

Reply:
It could be. I just figured because only this test needs it, I might as well just add it here, but I suppose if other tests might need it in the future it would be nice to have it in `setUp`. I'll move it.